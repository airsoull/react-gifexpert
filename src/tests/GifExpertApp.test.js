import React from 'react';

const { shallow } = require('enzyme');
const { default: GifExpertApp } = require('../GifExpertApp');

describe('test for GifExpertApp component', () => {

  const wrapper = shallow(<GifExpertApp />);

  test('should show the component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should show a list of categories', () => {
    expect(wrapper.find('GifGrid').length).toBe(3);
  });
});
