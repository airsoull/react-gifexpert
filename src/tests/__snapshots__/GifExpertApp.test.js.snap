// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for GifExpertApp component should show the component correctly 1`] = `
<Fragment>
  <h2>
     GifExpert! 
  </h2>
  <AddCategory
    placeholderText="Search yours GIFs..."
    setCategories={[Function]}
  />
  <hr />
  <GifGrid
    category="Hunter X Hunter"
    key="Hunter X Hunter"
  />
  <GifGrid
    category="One Piece"
    key="One Piece"
  />
  <GifGrid
    category="Boku no hero academia"
    key="Boku no hero academia"
  />
</Fragment>
`;
